MAUI/CheckBox/Getting-Started.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: post
+title: Getting Started with .NET MAUI CheckBox | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI CheckBox (SfCheckBox) control, its elements and more.
+platform: Maui
+control: SfCheckBox
+documentation: ug
+---
+
+# Getting Started with .NET MAUI CheckBox
+
+This section explains the steps required to work with the `.NET MAUI CheckBox` control.
+
+## Creating an application with .NET MAUI CheckBox
+
+This section explains the steps required to work with the `CheckBox` control for .NET MAUI.
+
+## Adding a .NET MAUI CheckBox reference
+
+Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add `.NET MAUI CheckBox` to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.Buttons`, and install it.
+
+## Handler registration 
+
+In the MauiProgram.cs file, register the handler for the Syncfusion core.
+
+{% highlight c# hl_lines="6 17" %}
+
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+
+namespace ButtonSample
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }      
+    }
+}   
+
+{% endhighlight %}  
+
+## Create a Simple .NET MAUI SfCheckBox
+
+The [.NET MAUI CheckBox] (SfCheckBox) control is configured entirely in C# code or by using XAML markup. The following steps explain how to create a [`SfCheckBox`] and configure its elements.
```

Comment:
The [.NET MAUI CheckBox] (SfCheckBox) control can be configured entirely in C# code using XAML markup. The following steps explain how to create a [`SfCheckBox`] and configure its elements.